jekyll/_config.yml

```
@@ -78,7 +78,7 @@ asciidoc:
       terraformversion: 0.15.4
       kubectlversion: 1.19
       helmversion: 3.4.0
-      kotsversion: 1.47.3
+      kotsversion: 1.65.3
```

There is no x.y.3 out.
```suggestion
      kotsversion: 1.65.0
```

---

ah thank you! accident